constants/Bingo.json

```
@@ -23,101 +25,112 @@
-        "Wear armor enchanted with Growth 5 (Enchanting Level 5).",
+        "Wear armor enchanted with Growth 5 ",
+        "(Enchanting Level 5).",
         "Applying Growth 5 costs 50 Levels each."
       ]
     },
     "Exterminator": {
       "difficulty": "Hard",
       "note": [
-        "Kill farm animals in the Barn with a Cleaver with Cleave 5 and Sharpness 5",
-        "or spiders in Arachne's Burrow (-309 92 -209) in the Spider's Den with a Dreadlord Sword.",
-        "You can also kill Endermen in The End to get progress towards the Lucky Looter ",
+        "Kill farm animals in the Barn with a Cleaver",
+        "with Cleave 5 and Sharpness 5",
+        "or spiders in Arachne's Burrow (-309 92 -209)",
+        "in the Spider's Den with a Dreadlord Sword.",
+        "You can also kill Endermen in The End to",
+        "get progress towards the Lucky Looter",
         "and Enderman Slayer Community Goal."
       ]
     },
     "End Stone Collector": {
       "difficulty": "Easy",
       "note": [
-        "Go to The End to mine Ender Nodes and kill Nest Mites for Enchanted End Stone.",
+        "Go to The End to mine Ender Nodes and",
+        "kill Nest Mites for Enchanted End Stone.",
         "You need to be Combat Level 12 to go there."
       ]
     },
     "Dwarves Defender": {
       "difficulty": "Medium",
       "note": [
-        "Kill Goblins in the Goblin Burrows (-100 163 150) while an event is going on.",
+        "Kill Goblins in the Goblin Burrows (-100 163 150)",
+        "while an event is going on.",
         "You need Mining Level 12 to visit the required island."
       ]
     },
     "A Lost Cause": {
       "difficulty": "Medium",
       "note": [
         "Start an Entrance floor and play Mage/Tank/Berserker to use",
-        "the classes abilities to kill a Lost Adventurer. You only need to get the last hit",
+        "the classes abilities to kill a Lost Adventurer.",
+        "You only need to get the last hit",
         "on it to count, so you can get help from your team.",
         "You need Combat Level 15 to join a Dungeon."
       ]
     },
     "Dark Oak Wood Collector": {
       "difficulty": "Hard",
       "note": [
-        "Optimal for this would be using a Golden Axe with Efficiency 5 and",
-        "having the Haste 3 potion effect to chop down Dark Oak. Reforge your Axe to Double-Bit or Lush",
-        "for bonus foraging fortune. Unlock Dark Oak Minions and place them on your island.",
-        "Remember to level them up using the material they produce and give them at least a medium storage",
-        "and the best fuel you can afford if you can collect them in time.",
-        "You can also manually forage in the Dark Thicket area (-353 103 -23) in The Park (/warp park)."
+        "Optimal tool:",
+        "Golden Axe with Efficiency 5",
+        "Reforges: Double-Bit or Lush",
+        "Get Haste 3 from a splash to instamine wood",
+        "forage in the Dark Thicket (-353 103 -23)",
```

```suggestion
        "Forage in the Dark Thicket (-353 103 -23)",
```

Also, PortalHub has a Dark Thicket portal, maybe mention that?